tox.ini

```diff
@@ -8,6 +8,8 @@ extras =
     test
     idna: idna
 deps =
+    # Use bcrypt on some, but not all envs
+    py38: bcrypt~=3.1; sys_platform=="linux"
```

Comment:
We should talk a bit about how we want to handle optional deps here. Do we want to introduce an extra, e.g. `cryptography[ssh]`?

Reply:
It is quite rare case, not sure install options are worth the hassle.

And it should not be [ssh], as nonencrypted keys work fine, but perhaps [all], [with-optional-deps], [with-everything], [with-extras] or something like that to be ready for future potential extra deps. If there will be install-options, it would be better to limit those to only single one.

Although the best solution is to merge bcrypt (and argon2) to cryptography as standard building blocks...  They are used frequently enough that having them in single place would be good idea.

Reply:
I'm sorry, I answered without looking into setup.py, there are already several install options. In that case the [ssh] is fine.

Still if it is possible for bcrypt to be part of cryptography itself in the future, then going with hard-dep is perhaps fine?  As the bcrypt is small and it's deps are much simpler than for cryptography itself?

Reply:
I'm not super excited about pulling more chunks into cryptography directly. I'd rather try to convince OpenSSL to support eks blowfish for this specific use case 😁, but I suppose that's a problem for another day.

@alex do you have an opinion on hard dep here? `bcrypt` is pretty self-contained and there's obviously a simplicity advantage in just hard depending on it, but it does feel a bit weird.